### PR TITLE
reset epoch for oct 12th

### PR DESCRIPTION
**Pull request by vishalchangrani**

### CODE REVIEW

File: transactions/reset-epoch/2023/oct-12/arguments.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[
+	{
```
Comment (vishalchangrani) [inline transactions/reset-epoch/2023/oct-12/arguments.json:2]: this is tbd

**Review by bjartek — APPROVED** [review]

File: transactions/reset-epoch/2023/oct-12/arguments.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		"type":"UInt64",
+		"value":"80"
+	},
+	{
+		"type":"String",
+		"value":"f28339976b22b61917e69fb5eca9eeb8"
+	},
+	{
+		"type":"UInt64",
+		"value":"0"
```
Comment (vishalchangrani) [inline transactions/reset-epoch/2023/oct-12/arguments.json:12]:
```suggestion
		"value":"100000000"
```
100M (current view is 7M)

File: transactions/reset-epoch/2023/oct-12/arguments.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	},
+	{
+		"type":"UInt64",
+		"value":"426399"
```
Comment (vishalchangrani) [inline transactions/reset-epoch/2023/oct-12/arguments.json:16]:
```suggestion
		"value":"200000000"
```
200M

File: transactions/reset-epoch/2023/oct-12/arguments.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	},
+	{
+		"type":"UInt64",
+		"value":"482999"
```
Comment (vishalchangrani) [inline transactions/reset-epoch/2023/oct-12/arguments.json:20]:
```suggestion
		"value":"300000000"
```
300M

**Review by vishalchangrani — COMMENTED** [review]
This changes the view values to be far enough above current views that they will never occur before the next spork. This means that no epoch-related events will occur for the duration of the reset epoch, and allows us to increment the epoch counter for each week until the next spork when the network leaves epoch fallback mode.

File: transactions/reset-epoch/2023/oct-12/arguments.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+[
+	{
+		"type":"UInt64",
+		"value":"80"
```
Comment (joshuahannan) [inline transactions/reset-epoch/2023/oct-12/arguments.json:4]:
```suggestion
		"value":"96"
```

**Pull request merged**